build.sh

```diff
@@ -401,6 +410,15 @@ build_one() {
                 INCLUDE_PATH="${root}/include" \
                 LIBRARY_PATH="${root}/lib" install
         ;;
+    libzip)
+	do_cmake \
```

Comment:
autoconf gives me a compile error. "Making all in regress
make[2]: Entering directory '/home/markus/source/openslide-winbuild/32/build/libzip-1.1.3/regress'
/bin/sh ../libtool  --tag=CC   --mode=compile i686-w64-mingw32-gcc -static-libgcc -DHAVE_CONFIG_H -I. -I..  -I../lib -I../lib -I../src -D_FORTIFY_SOURCE=2 -I/home/markus/source/openslide-winbuild/32/root/include  -O2 -g -mms-bitfields -fexceptions -ftree-vectorize -msse2 -mfpmath=sse -mstackrealign -c -o malloc.lo malloc.c
libtool: compile:  i686-w64-mingw32-gcc -static-libgcc -DHAVE_CONFIG_H -I. -I.. -I../lib -I../lib -I../src -D_FORTIFY_SOURCE=2 -I/home/markus/source/openslide-winbuild/32/root/include -O2 -g -mms-bitfields -fexceptions -ftree-vectorize -msse2 -mfpmath=sse -mstackrealign -c malloc.c  -DDLL_EXPORT -DPIC -o .libs/malloc.o
malloc.c:39:19: fatal error: dlfcn.h: No such file or directory
 #include <dlfcn.h>
"

Also, a libzip maintainer said "The intention was to provide CMake for Windows and let the other systems that can deal with it use the configure script"
https://nih.at/listarchive/libzip-discuss/msg00634.html